packages/ui-kit/src/components/Logger/LogProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext } from 'react'
+
+import { LogLevels } from './Log.types'
+
+export type LogContextProps = {
+  logLevel?: LogLevels
+}
+
+export const LogContext = createContext<LogContextProps | undefined>(LogLevels.Error)
+
+export type LogProviderProps = {
+  children?: React.ReactNode
+  logLevel?: LogLevels
```

Comment:
How about documenting this?
```suggestion
  /** The log level to use (defaults to "error"). */
  logLevel?: LogLevel
```